README.md

```diff
@@ -256,6 +303,10 @@ ORCHESTRATION_USER
 ###############################################################################
 WORKLOAD_IMAGE
 
+WORKLOAD_JOB_NODE_SELECTOR
+WORKLOAD_JOB_TAINT
+WORKLOAD_JOB_PRIVILEGED
```

Comment:
I think these are accidental, the http workload actually doesn't have these vars (https://github.com/openshift-scale/workloads/blob/master/workloads/vars/http.yml)

Reply:
Ah, might have left them during the rebase. Fixed, thanks.